src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -132,7 +132,8 @@ uct_rc_mlx5_iface_poll_tx(uct_rc_mlx5_iface_common_t *iface)
     unsigned qp_num;
     uint16_t hw_ci;
 
-    cqe = uct_ib_mlx5_poll_cq(&iface->super.super, &iface->cq[UCT_IB_DIR_TX]);
+    cqe = uct_ib_mlx5_poll_cq(&iface->super.super, &iface->cq[UCT_IB_DIR_TX],
+                              /*is_rx*/ 0);
```

Comment:
when we see a compressed CQE we can tell if it's RX or TX by the opcode. so no need to pass this arg.
anyway, the decompression is expensive, so rx/tx branch is not significant IMO